cryodrgn/commands/preprocess.py

```diff
@@ -44,6 +51,10 @@ def warnexists(out):
         log(f'Warning: {out} already exists. Overwriting.')
 
 def main(args):
+    if cp is None and args.use_cupy:
```

Comment:
My recommendation would be to raise a RuntimeError if `cp is None`. Silently changing things in `args` will make it tricky to test the code against different scenarios (we plan on extending the tests soon hopefully). I'm not sure if we do this in other places too, but I think it should be avoided.